cogs/spawning.py

```diff
@@ -236,6 +236,11 @@ async def on_message(self, message: discord.Message):
             self.spawn_threshold *= 1.1
 
     async def spawn_pokemon(self, channel, species=None, incense=None):
+        prev_species = ""
```

Comment:
Last nitpick, I think it'd be better to have a `None` here instead of empty string. Otherwise, looks good!

Reply:
Changed